dependencies.gradle

```diff
@@ -9,3 +9,7 @@ ext.buildLibs = [
 ext.configs = [
         androidLibrary: "${project.rootDir}/build-configuration/android-library.gradle",
 ]
+
+ext.testLibs = [
+        turbine: "app.cash.turbine:turbine:0.7.0"
+]
```

Comment:
This allows us to test flow emissions that happen during the runtime of a given block of logic.